client/src/components/DataFiles/DataFilesModals/DataFilesMkdirModal.js

```diff
@@ -66,7 +69,7 @@ const DataFilesMkdirModal = () => {
         >
           <Form>
             <ModalHeader toggle={toggle}>
-              Creating folder in: {params.path || '/'}
+              Creating folder in {currentSystem.name}: {params.path || '/'}
```

Comment:
This will likely break for Shared Workspaces/Projects since those systems aren't being pulled into systems.systemList reducer.

Reply:
Can the form of this text be changed to match the DataFilesUploadModal where it is `Uploading to {systemDisplayName}/{params.path}` (i.e. no space or colon)